src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -257,7 +257,7 @@ enum
     DDIX_FILTER_NONSELECTED,
 };
 
-struct list_item
+struct ListItem
```

Comment:
Could be more descriptive, like `ScenarioListItem`:
```suggestion
struct ObjectListItem
```